LDM-602.tex

```diff
@@ -0,0 +1,646 @@
+\documentclass[SE,toc,lsstdraft]{lsstdoc}
+
+% We use commands to make it easy to find where parameter names and units
+% are defined in the tables, and to allow hyphenation.
+\newcommand{\paramname}[1]{\hspace{0pt}#1}
+\newcommand{\unitname}[1]{\hspace{0pt}#1}
+
+\setcounter{secnumdepth}{5}
+
+%% Retrieve date and model version
+\setDocUpstreamLocation{MagicDraw SysML}
+\setDocUpstreamVersion{72}
+
+\date{2018-01-16}
+
+%% Allow arbitrary latex to be inserted at the end of the document.
+%% Define a new version of this command in metadata.tex. It will
+%% be run before the references are displayed.
+\newcommand{\addendum}{}
+
+%% Define the document title, authors, handle, and change record
+\input metadata.tex
+
+% Environment for displaying the parameter tables in
+% a consistent manner. No arguments as there are no
+% captions or labels.
+\newenvironment{parameters}[0]{%
+\setlength\LTleft{0pt}
+\setlength\LTright{\fill}
+\begin{small}
+\begin{longtable}[]{|p{0.49\textwidth}|l|p{0.6in}|p{1.70in}@{}|}
+
+\hline \textbf{Description} & \textbf{Value} & \textbf{Unit} & \textbf{Name} \\ \hline
+\endhead
+
+\hline \multicolumn{4}{r}{\emph{Continued on next page}} \\
+\endfoot
+
+\hline\hline
+\endlastfoot
+}{%
+\hline
+\end{longtable}
+\end{small}
+}
+
+\begin{document}
+\maketitle
+
+\section{\citeds{LSE-61} Flowdown}
+
+This section contains direct copies of requirements from \citeds{LSE-61}, based on the Level 1 requirements defined in \citeds{LDM-148}.
+
+\subsection{Level-1 Production Completeness}
+
+\label{DMS-L1-REQ-0025}
```

Comment:
It's seems like we're crying out for a macro to make defining requirements easy, rather than using all this explicit label/emph/bf/etc markup. But this is at least consistent with what's happening in other documents, I suppose.

Reply:
I can definitely use macros but since no human ever edits this content it doesn't gain us much. You can see the template at https://github.com/lsst-se/MagicDraw-macros/tree/master/docgen_templates/latex

The standard is to put requirements in numbered sections so a macro is a little tricky in the sense that the obvious approach of `\requirement{title}{id}{spec}` can't work.